Sources/Networking/Responses/OfferingsResponse.swift

```diff
@@ -54,7 +54,70 @@ struct OfferingsResponse {
     let placements: Placements?
     let targeting: Targeting?
     let uiConfig: UIConfig?
+    let config: Config?
 
+    public struct Config {
```

Comment:
I added a `Config` 'envelope' here in order to future-proof ourselves for more configuration values. But we can also just call it `priceFormattingRuleSets` and only have this single purpose field here, happy to go either way